extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -220,6 +220,22 @@ export class AmpSlideScroll extends BaseSlides {
     if (slide !== undefined) {
       this.showSlideWhenReady_(slide);
     }
+    const slideCount = mutations['slide-count'];
+    if (slideCount !== undefined && isFinite(slideCount)) {
+      let slideCountNum = Number(slideCount);
```

Comment:
you could do 

const slideCount = Number(mutations['slide-count']);
if (isFinite(slideCount))